test/boostTest.cpp

```diff
@@ -83,6 +83,17 @@ int main( int argc, char* argv[] )
 {
 	//Initialize options
 	dev::test::Options const& opt = dev::test::Options::get(argc, argv);
+
+	char* path = (argv[0]);
+	argc = 3;
```

Comment:
the only way I found it work without issues. It is just to test on travis only this test suite. 
will not be included to the final PR